packages/app-project/stores/Collections.js

```diff
@@ -0,0 +1,108 @@
+import { autorun } from 'mobx'
+import { addDisposer, flow, getRoot, types } from 'mobx-state-tree'
+import auth from 'panoptes-client/lib/auth'
+import asyncStates from '@zooniverse/async-states'
+
+export const Collection = types
+  .model('Collection', {
+    display_name: types.maybeNull(types.string),
+    favorite: false,
+    id: types.identifier,
+    links: types.maybeNull(types.frozen({})),
+    private: true
+  })
+
+const Collections = types
+  .model('Collections', {
+    error: types.maybeNull(types.frozen({})),
+    favourites: types.maybeNull(Collection),
+    loadingState: types.optional(types.enumeration('state', asyncStates.values), asyncStates.initialized),
+  })
+
+  .actions(self => {
+    let client
+
+    function createProjectObserver () {
+      const projectDisposer = autorun(() => {
+        const project = getRoot(self).project
+        const user = getRoot(self).user
+        if (project.id && user.id) {
+          self.fetchFavourites()
+        }
+      })
+      addDisposer(self, projectDisposer)
+    }
+
+    return {
+      afterAttach () {
+        client = getRoot(self).client.collections
+        createProjectObserver()
+      },
+
+      createFavourites: flow( function * createFavourites () {
+        const project = getRoot(self).project
+        self.loadingState = asyncStates.loading
+        const token = yield auth.checkBearerToken()
+        const authorization = `Bearer ${token}`
+        const data = {
+          display_name: `Favourites ${project.slug}`,
```

Comment:
Are we defaulting to British English? Should this be using a translation string?

Reply:
Im not fussed. PFE uses ‘Favorites’ and volunteers can edit the name anyway. 

Localisation would be a nice little touch.